### PR TITLE
Shuai_Yuan_Hw_1

### PR DESCRIPTION
**Pull request by shuaify**
The final version of Shuai Yuan homework 1. Work flow can be seen clearly from each commit and push. 
I think the hardest part for me is inserting images. I googled several methods from stackoverflow and finally made it. At the beginning I just forgot to push the local photo in the repository. 
In addition, I made a very stupid mistake: I forgot to push every commit I did and I couldn't find where all the changes I have made were going. And finally I recalled this and finish the homework on time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+# Homework One 
+##**Basic Information**
```
Comment (guivann) [inline README.md:2]: There should be a space after the pound, and this also applies to line 14, 18, and 32.

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+  
+Department:*Harris School of Public Policy*
+  
+Graduation Year:*2020*
+  
+E-mail:*shuaiyuan@uchicago.edu*
+
+##**Image**
+  
+![Shuai Yuan 2016.](C:/Users/Admin/Desktop/Computing for Social Science/Shuai Yuan-hw1/Shuai Yuan hw-1/Shuai Yuan.png)
```
Comment (guivann) [inline README.md:16]: This directory only works on your own computer. I see that you already uploaded the image to GitHub and, thus, I believe this should work:
`Shuai Yuan 2016.`

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+
+##**Coursework Relvant to R**##
```
Comment (guivann) [inline README.md:18]: Typo

**Review by guivann — CHANGES_REQUESTED** [review]
I can definitely tell how much effort you put in. I like the multi-layer lists. Some minor changes should make this introduction perfect.

File: README.md (hunk 2 above)
Comment (bensoltoff) [reply README.md:16]: Correct. We'll talk in future weeks about best practices for identifying the locations of files

File: README.md [hunk 4]
```diff
@@ -1 +1,34 @@
...
+
+* 1.Statistics I: Basics
```
Comment (bensoltoff) [inline README.md:20]: You can actually make use of numbered lists in Markdown to format this list

```
1. Statistics I: Basics
    + Probability Theory
    + Hypothesis Testing 
        - Doing t-tests
1. Statistics II: Regression Analysis
    + Multivariant Regression 
        - Interaction    
    + Logistical Regression
    + Difference in Difference
    + Instrumental Variable
```

1. Statistics I: Basics
    + Probability Theory
    + Hypothesis Testing 
        - Doing t-tests
1. Statistics II: Regression Analysis
    + Multivariant Regression 
        - Interaction    
    + Logistical Regression
    + Difference in Difference
    + Instrumental Variable

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |           |        x      |            |
| **Ease of access**          |           |       x       |            |

**Pull request closed without merging**